pika/adapters/twisted_connection.py

```diff
@@ -183,126 +276,30 @@ def single_argument(*args):
 
             try:
                 method(*args, **kwargs)
-            # TODO this except without types would suppress system-exiting
-            # exceptions, such as SystemExit and KeyboardInterrupt. It should be
-            # at least `except Exception` and preferably more specific.
-            except:
+            except Exception:  # pylint: disable=W0703
                 return defer.fail()
             return d
 
         return wrapped
 
-    def __clear_consumer(self, ret, queue_name):
-        self.__consumers.pop(queue_name, None)
+    def _clear_consumer(self, ret, queue_name):
+        self._consumers.pop(queue_name, None)
         return ret
 
-    def __clear_call(self, ret, d):
-        self.__calls.discard(d)
+    def _clear_call(self, ret, d):
+        self._calls.discard(d)
         return ret
 
     def __getattr__(self, name):
```

Comment:
@abompard, are you planning to add the remaining explicit Channel methods?

Reply:
I didn't think so, I thought you meant explicit signatures for the redefined methods only. But if you prefer not using `__getattr__`, I can copy all the methods in `WRAPPED_METHODS` in this class.
I'm just a bit worried it will be more maintenance in the long run, since they will have to be adapted in this class too whenever they are changed in the main Channel class. But I'll follow you if you think it's worthwhile.